src/index.js

```diff
@@ -47,8 +68,8 @@ class LeaderBoard extends React.Component {
             <tr>
               <th>#</th>
               <th>Camper Name</th>
-              <th>Points in 30days</th>
-              <th>All time points</th>
+              <th className="link" onClick={this.recentPoints.bind(this)}>Points in 30 days</th>
```

Comment:
Binding can be done in the constructor. If done like this, the binding will be done during each render which is unnecessary.